doc/appendices/administration.rst

```diff
@@ -0,0 +1,148 @@
+.. Copyright 2022 UT-Battelle, LLC, and other Celeritas developers.
+.. See the doc/COPYRIGHT file for details.
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _administration:
+
+**************
+Administration
+**************
+
+This appendix includes administrative details describing the roles and
+responsibilities of participants.
+
+
+Community standards
+===================
+
+Be respectful. Communicate openly. Resolve disagreements peacefully.
+
+
+Resolution process
+------------------
+
+If a disagreement about Celeritas can't be immediately resolved, it's OK to
+postpone resolution (maybe you need more data, more time to think, or it's not
+important enough at the end of the day). In the case that a resolution is
+needed quickly (e.g., if it's a blocking issue), it should be discussed among
+the :ref:`core team <roles>` until an agreement is reached. If the issue is
+still contentious, the leadership team should make an executive decision,
+giving weight to the appropriate capability area lead if possible.
+
+
+.. _roles:
+
+Roles
+=====
+
+The roles of the Celeritas code base are related to the roles
+set out in the SciDAC project management plan (PMP). These roles should be
+reflected in the different teams and/or access restrictions in GitHub.
+
+Leadership team member
+   The "leadership team" comprises the PIs and capability leads in the PMP who
+   work directly on the Celeritas code. They are responsible for
+   the long-term and big-picture project goals, and they ensure that project
+   milestones are met.
+
+Code lead
+   The code lead is the "core capability lead" described in the PMP and
+   is responsible for steering the technical implementation of the codebase to
+   meet the long-term project goals. One key responsibility of the code lead is
+   to set up milestones for releases and execute the release process described
+   below.
+
+Maintainer
+   Maintainers should be familiar with most or all parts of the codebase and
+   are responsible for merging pull requests and closing issues.
+
+Core team member
+   The "core team" are those currently funded to work on Celeritas. Core team
+   members are responsible for reviewing pull requests in accordance with the
+   :ref:`contributing guidelines <contributing>`. They should regularly
+   contribute new code or publish new results using Celeritas.
```

Comment:
I feel like this might be @sethrj 's way of kicking me off the core team 🤣

Reply:
I modified this slightly... but since there's no requirement that leadership must be "core", it's OK if you're not one :) as the later statement says, everyone's contributions are recognized.